server/js/sockets.js

```diff
@@ -172,10 +176,20 @@ sockets.init = function socketInit(server) {
           payloadString,
         );
 
-        // store last received payload for device globally
+        // Store last received payload for device globally
         global.lastRecordingPayloads[
           topicString[topicString.length - 1]
         ] = payloadString;
+      } else if (
+        topicString.slice(0, -1).join('/') === '/v3/camera/video-feed/status'
+      ) {
+        // Store last received payload for device globally
+        global.lastVideoFeedPayloads[
+          topicString[topicString.length - 1]
+        ] = payloadString;
```

Comment:
`payloadString` here is serialized on the other end with a `JSON.parse`. This could be avoided by stashing the `payload` object instead.

Reply:
Done
5312dd9

Reply:
`payloadString` is derived from payload so there's no need to parse it again!
```suggestion
        ] = payload;
```

Reply:
`payload` is not a JSON type object but seems to be an `ArrayBuffer`. Not sure if there is some method to call than can change it to JSON without parsing it as I couldn't the docs for `mqttClient.on`.